base/server/python/pki/server/__init__.py

```
@@ -1863,6 +1863,20 @@ def install_fapolicy_rules(self):
             mode=mode,
             force=True)
 
+        logger.info('Adding JSP and template files to fapolicy trust database')
+
+        for root, dirs, files in os.walk(PKIServer.SHARE_DIR):
+            for filename in files:
+                if filename.endswith('.jsp') or filename.endswith('.template'):
+                    filepath = os.path.join(root, filename)
+                    cmd = ['fapolicyd-cli', '--file', 'add', filepath]
+                    logger.debug('Command: %s', ' '.join(cmd))
+                    subprocess.call(cmd, stderr=subprocess.DEVNULL)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)